### PR TITLE
Gen 3427 kirkstone migration update

### PR DESCRIPTION
**Pull request by valeria1995**
Update meta-openssl102 layer to kirkstone

### CODE REVIEW

File: recipes-connectivity/openssl/openssl_1.0.2u.bb [hunk 1]
```diff
@@ -329,18 +329,18 @@ do_install_ptest () {
...
-FILES_${PN}-misc = "${libdir}/ssl/misc"
-FILES_${PN} =+ "${libdir}/ssl/*"
-FILES_${PN}_append_class-nativesdk = " ${SDKPATHNATIVE}/environment-setup.d/openssl.sh"
+FILES:libcrypto = "${libdir}/libcrypto${SOLIBS}"
+FILES:libssl = "${libdir}/libssl${SOLIBS}"
+FILES:openssl-conf = "${sysconfdir}/ssl/openssl.cnf ${libdir}/ssl/openssl.cnf"
+FILES:${PN}-engines = "${libdir}/ssl/engines/*.so ${libdir}/engines"
+FILES:${PN}-misc = "${libdir}/ssl/misc"
+FILES:${PN} =+ "${libdir}/ssl/*"
+FILES:${PN}:append:class-nativesdk = " ${SDKPATHNATIVE}/environment-setup.d/openssl.sh"
```
Comment (diego-santacruz) [inline recipes-connectivity/openssl/openssl_1.0.2u.bb:338]: None of these FILES_xx variables are overrides as far as I can see, so no replacing of `_` by `:`, only `_append_class-nativesdk` is a double override and needs replacing.

File: recipes-connectivity/openssl/openssl_1.0.2u.bb [hunk 2]
```diff
@@ -329,18 +329,18 @@ do_install_ptest () {
...
 
-CONFFILES_openssl-conf = "${sysconfdir}/ssl/openssl.cnf"
+CONFFILES:openssl-conf = "${sysconfdir}/ssl/openssl.cnf"
```
Comment (diego-santacruz) [inline recipes-connectivity/openssl/openssl_1.0.2u.bb:340]: Same here, these are not overrides as far as I can see.

File: recipes-connectivity/openssl/openssl_1.0.2u.bb [hunk 3]
```diff
@@ -329,18 +329,18 @@ do_install_ptest () {
...
 
-RRECOMMENDS_libcrypto += "openssl-conf"
-RDEPENDS_${PN}-misc = "${@bb.utils.filter('PACKAGECONFIG', 'perl', d)}"
-RDEPENDS_${PN}-ptest += "${PN}-misc make perl perl-module-filehandle bc"
+RRECOMMENDS:libcrypto += "openssl-conf"
+RDEPENDS:${PN}-misc = "${@bb.utils.filter('PACKAGECONFIG', 'perl', d)}"
+RDEPENDS:${PN}-ptest += "${PN}-misc make perl perl-module-filehandle bc"
```
Comment (diego-santacruz) [inline recipes-connectivity/openssl/openssl_1.0.2u.bb:344]: Same here, these are not overrides as far as I can see.

**Review by diego-santacruz — CHANGES_REQUESTED** [review]
I think many of the replacements of `_` and `:` are not for overrides and thus it is not correct.

Check that the packaging output (list of files in each rpm, scripts, etc.) are the same between the build of this recipe in dunfell and kirkstone.

File: recipes-connectivity/openssl/openssl_1.0.2u.bb (hunk 3 above)
Comment (valeria1995) [reply recipes-connectivity/openssl/openssl_1.0.2u.bb:344]: As discussed earlier, there is the documentation for kirkstone: [kirkstone variables](https://docs.yoctoproject.org/4.0.14/ref-manual/variables.html).

The right syntax will be:
- FILES:xxx
- CONFFILES:xxx
- RRECOMMENDS:xxx
- RDEPENDS:xxx

**Review by diego-santacruz — APPROVED** [review]
All good indeed, I was mistaken. Sorry for the noise.

**Pull request merged**